dnq/src/main/kotlin/kotlinx/dnq/util/TransientEntityUtil.kt

```diff
@@ -115,4 +116,10 @@ fun XdEntity.reattachAndGetLink(linkName: String): Entity? {
     return reattach(session).getLinkEx(linkName, session)
 }
 
-val XdEntity.isReadOnly: Boolean get() = (entity as TransientEntity).isReadonly
+val XdEntity.isReadOnly: Boolean
+    get() {
+        if (entity is OVertexEntity) {
+            throw IllegalStateException("LOshadinaya zalupa")
```

Comment:
Leonid I will kill you

Reply:
Oh, just to many merge requests made me a bit blind on changes i commit in little fixes. Sorry

Reply:
I do not believer a word :-)))